buster-amd64/mkrescue

```diff
@@ -60,3 +63,5 @@ tar cvf ${OUTPUT}/wmirescue_${ARCH}-netboot.tar -C ${OUTPUT}/netboot .
 ./bin/dist_ipxe ${VERSION} ${OUTPUT} --nofirmware
 ./bin/dist_iso ${VERSION} ${ARCH} ${OUTPUT}
 ./bin/dist_iso ${VERSION} ${ARCH} ${OUTPUT} --nofirmware
+
+umask ${UMASK_ORIG}
```

Comment:
No need to revert umask to orig value as this is the end of the file ;)

Reply:
Yeah, you are right. Removed.